prody/proteins/pdbfile.py

```diff
@@ -162,6 +162,10 @@ def parsePDBStream(stream, **kwargs):
     elif model != 0:
         ag = AtomGroup(str(kwargs.get('title', 'Unknown')) + title_suffix)
         n_csets = 0
+    if "needfullname" in kwargs:
```

Comment:
Hi Wenzhi,

Thanks for the bug fixes! Because of some check errors in the requested branch (though it could be nothing), we cannot take the risk of merging it. Therefore I am redoing all your fixes. I want to ask if you can give some explanation on what this argument is doing? A few sentences is okay. 

Thanks,
She